clusterloader2/pkg/measurement/common/slos/api_responsiveness_prometheus.go

```diff
@@ -44,7 +44,7 @@ const (
 
 	currentAPICallMetricsVersion = "v1"
 
-	filters = `verb!~"WATCH|WATCHLIST|PROXY|CONNECT"`
+	filters = `verb!~"WATCH|WATCHLIST|PROXY|CONNECT", subresource!~"proxy"`
```

Comment:
Given you touch this, let's also remove: WATCHILIST, PROXY and CONNECT and live only WATCH here.

They don't seem to used sine 1.11.

Reply:
Done